packages/cdktf/lib/terraform-provisioner.ts

```diff
@@ -0,0 +1,380 @@
+import { ref } from "./tfExpression";
+import { Token } from "./tokens/token";
+import { snakeCase } from "./util";
+/**
+ * Most provisioners require access to the remote resource via SSH or WinRM and expect a nested connection block with details about how to connect.
+ *
+ * See {@link https://www.terraform.io/language/resources/provisioners/connection connection}
+ */
+export interface ISSHProvisionerConnection {
```

Comment:
I'd have gone with this, but no hard feelings here:
```suggestion
export interface ISshProvisionerConnection {
```

Reply:
just saw that "WinRM" is also lowercased to `Winrm.`